### ISSUE TITLE
Update Gitpod integration.

### ISSUE THREAD

**Issue by Decipher** [druxt/druxt.js#343]
**Is your feature request related to a problem? Please describe.**
As a project manager I want to ensure the Gitpod integration is up-to-date and easy to maintain.

**Describe the solution you'd like**
- [ ] Remove Drupal task; Drupal should only be started if required, and details should be provided in README
- [ ] Update DDev config to latest version; Remove unnecessary ports and such.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by Decipher** [comment]
Got stuck on this getting DDev integration back up and running correctly, so have deferred it for the moment.

The goal is to be able to have a copy of Drupal 9 and ContentaCMS available to be started within the Gitpod as required, but not running by default as they may not be required.

/cc @shaal If you have a chance to take a look I'd appreciate some help.

**Comment by shaal** [comment]
@Decipher how can I help?
Do you want Drupal to start by using URL parameters? or should it be a manual command only after the workspace is open?
In the upcoming branch of DrupalPod, I have a separate task that prepares Drupal installation, I create a ddev-snapshot (very fast), and upload it to Google cloud (for freeee, I think). During prebuild - I download that backup from Google cloud, so it's ready for the end user if needed. Is that what you're looking for or something else?

**Comment by Decipher** [comment]
Ideally I would like to have multiple code bases:
- Drupal 9
- Drupal 10
- ContentaCMS
- etc

I don't want them running by default, but able to be spun up easily as required, by running a script from within the relevant code base.

I stated updating the scripts to have the latest gitpod ddev config  alongside a committed ddev config.yaml, but it kept erroring.

**Comment by shaal** [comment]
Did you have a chance to look at these scripts?
https://github.com/shaal/DrupalPod/tree/ready-made-envs/.gitpod/drupal/envs-prep
